pkg/controller/appdefinition/testdata/volumes/contextdir/expected.yaml

```diff
@@ -45,6 +45,11 @@ spec:
       serviceAccountName: container-name
       containers:
         - name: container-name
+          resources:
```

Comment:
If there's no global default and the user hasn't set a memory value there should be no change to the output here.  There should be no "resources" field at all.